GCTC-NTGC/gc-digital-talent
# Copy -  4096 - Update homepage content

Resolves #4096 

## 📋 Summary

This updates the content for the "Check it out" section on the homepage.

## ✅ TO DO

- [x] Update "From concept to code" link once supplied

## 📷 Screenshot

<img width="1164" alt="Screen Shot 2022-10-24 at 2 14 49 PM" src="https://user-images.githubusercontent.com/4127998/197596637-a5e17e2d-13e5-4664-ab4a-069d06580d04.png">

## 🧪 Testing

1. Run storybook `npm run storybook --workspace=talentsearch`
2. Navigate to the "Home Page" story
3. Confirm the content has updated and links go to the proper location
    - **Note**: "From concept to code" will go to a 404 if you update `.htaccess`. This _should_ be solved in #4026 

> **Note**
> Testing instructions use storybook for ease. To do more thorough testing it is recommended tp update the `.htaccess` pointing to `frontend/talentsearch/dist`

https://github.com/GCTC-NTGC/gc-digital-talent/blob/04edc87f0ceb788b93ac7a2d37acde983e06e6aa/infrastructure/php-container/src/.htaccess#L123-L125
